quantum/quantum.c

```diff
@@ -1151,22 +1162,22 @@ void matrix_scan_quantum() {
 #    endif
 
 #    ifndef BACKLIGHT_ON_STATE
-#        define BACKLIGHT_ON_STATE 0
+#        define BACKLIGHT_ON_STATE 1
 #    endif
 
 void backlight_on(uint8_t backlight_pin) {
-#    if BACKLIGHT_ON_STATE == 0
-    writePinLow(backlight_pin);
-#    else
+#    if BACKLIGHT_ON_STATE == 1
```

Comment:
Why invert this block?

Reply:
Consistency, the rest of these checks are `== 1`.